invenio_rdm_records/services/services.py

```diff
@@ -129,6 +129,7 @@ def search_community_records(
     ):
         """Search for records published in the given community."""
         self.require_permission(identity, "read")
+
```

Comment:
minor: this is not in master <https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/services/services.py#L132> I think there is something weird going on the rebase.

Reply:
I'm confused as I see that in master and also in the link you sent :/